src/wp-includes/class-wp-autoloader.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class WP_Autoloader
+ *
+ * WordPress Class, Interface and Trait Autoloader
+ *
+ */
+class WP_Autoloader {
+
+	/**
+	 * @var array
+	 */
+	private static $_classmap;
+
+	/**
+	 * WP_Autoloader constructor.
+	 */
+	function __construct() {
+
+		if ( ! defined( 'DISABLE_CORE_AUTOLOADER' ) ) {
```

Comment:
That has from an earlier version I committed but before I pushed. See [the current version](/staylor/develop.wordpress/pull/2/files#diff-748e02cd742247bfc3117214bebb22e1R5).

Actually, be sure to read and comment [on the collective files changed](/staylor/develop.wordpress/pull/2/files) as opposed to looking at individual changesets because you might well comment on something that is moot, such as this.